src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+
+type Props = {
+  errorMessage: string;
+  onCloseErrorMessage: () => void;
+};
+
+export const ErrorNotification: FC<Props> = ({
+  errorMessage,
+  onCloseErrorMessage,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        { hidden: !errorMessage.length },
```

Comment:
```suggestion
        { hidden: !errorMessage },
```
Empty string is falsy